frontend/src/locales/en.json

```diff
@@ -26,8 +26,23 @@
     "community": "Community",
     "continue-to-registration": "Continue to registration",
     "current-community": "Current community",
+    "myContributions": "My contributions",
     "other-communities": "Other communities",
-    "switch-to-this-community": "Switch to this community"
+    "switch-to-this-community": "Switch to this community",
+    "writing": "Writing"
+  },
+  "contribution": {
+    "activity": "Activity",
+    "formText": {
+      "h3": "Your contribution to the common good",
+      "lastMonth": "For  <b>{monthAndYear}</b>, you can still submit <b>{creation}</b> GDD.",
```

Comment:
Is this needed? Looks the same as `thisMonth`

Reply:
thisMonth is deleted an lastMonth is new **openAmountForMonth**